docker/openstack-client-centos/README.md

```diff
@@ -20,45 +20,47 @@ systemctl enable docker
 systemctl restart docker
 ```
 
-
 ## Running
 
-The process of creating and running the docker container is facilitated through the ```run.sh``` script inside this repository.  
+There are two options for running the `control-node` containers:
+
+1. Raw `docker run` command
+2. Background container via `docker-compose`
+
+### Docker Compose (recommended)
 
-It will produce the docker image based on a *Dockerfile* and run the docker container based on the following parameters:
+Starting the container is done with the following:
 
 ```
-$ ./docker/openstack-docker-client/run.sh --help
-
-     Usage: ./docker/openstack-docker-client/run.sh [options]
-     Options:
-     --configdir=<configdir>       : Directory containing Openstack configuration files (Default: ~/.openstack/)
-     --name=<name>                 : Name of the assembled image (Default: rhc-openstack-client)
-     --keep                        : Whether to keep the the container after exiting
-     --ssh=<ssh>                   : Location of SSH keys to mount into the container (Default: ~/.ssh)
-     --repository=<repository>     : Directory containing a repository to mount inside the container
-     --help                        : Show Usage Output
+cd ./docker/control-node-openstack
```

Comment:
this doesn't look correct - shouldn't it be `docker/openstack-client-centos`? ... or did you mean to rename it to `control-node-openstack`?